rust/datafusion/src/logical_plan/mod.rs

```diff
@@ -1264,31 +1050,6 @@ mod tests {
         Ok(())
     }
 
-    #[test]
-    fn test_visitor() {
-        let schema = Schema::new(vec![]);
-        assert_eq!("[]", format!("{}", display_schema(&schema)));
-    }
-
-    #[test]
-    fn test_display_empty_schema() {
-        let schema = Schema::new(vec![]);
-        assert_eq!("[]", format!("{}", display_schema(&schema)));
-    }
-
-    #[test]
-    fn test_display_schema() {
-        let schema = Schema::new(vec![
-            Field::new("id", DataType::Int32, false),
-            Field::new("first_name", DataType::Utf8, true),
-        ]);
-
-        assert_eq!(
-            "[id:Int32, first_name:Utf8;N]",
-            format!("{}", display_schema(&schema))
-        );
-    }
```

Comment:
Maybe one of the tests disapeared in this commit? (I count -3 +2)

Reply:
Good 👁️ 

I removed 
```
    fn test_visitor() {
        let schema = Schema::new(vec![]);
        assert_eq!("[]", format!("{}", display_schema(&schema)));
    }
```

Which is redundant, except for the name, with the test immediately below: 
```
    #[test]
    fn test_display_empty_schema() {
        let schema = Schema::new(vec![]);
        assert_eq!("[]", format!("{}", display_schema(&schema)));
    }
```

Aka it was removing a copy / paste bug (that I added 😞 )